src/main/java/mate/academy/spring/validation/PasswordValidator.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.spring.validation;
+
+import java.util.Objects;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import mate.academy.spring.dto.request.UserRequestDto;
+
+public class PasswordValidator implements ConstraintValidator<Password, UserRequestDto> {
+    private static final int MIN_PASSWORD_LENGTH = 64;
```

Comment:
isn't it too much??

Reply:
Indeed. When I wrote that number, my thoughts were on the maximum, not the minimum... 😂